doc/nrf/known_issues.rst

```diff
@@ -2026,13 +2034,43 @@ SoftDevice Controller
 
 In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.
 
```

Comment:
Another known issue to be added:
rst-class::  v2-2-0 v2-1-2 v2-1-1 v2-1-0

DRGN-18247: Assertion with :c:enumerator:`MPSL_CLOCK_HF_LATENCY_BEST`
  When setting  for the ramp-up time of the high-frequency crystal oscillator with :c:enumerator:`MPSL_CLOCK_HF_LATENCY_BEST`, it has been seen causes an assert in MPSL.

  **Workaround:** Use :c:enumerator:`MPSL_CLOCK_HF_LATENCY_TYPICAL` instead of :c:enumerator:`MPSL_CLOCK_HF_LATENCY_BEST` when setting for the time it takes for the HFCLK to ramp up.